AMBuildScript

```diff
@@ -266,31 +280,35 @@ class ExtensionConfig(object):
       '_stricmp=strcasecmp',
       '_snprintf=snprintf',
       '_vsnprintf=vsnprintf',
-      'typeof=__typeof__',
       'HAVE_STDINT_H',
       'GNUC',
     ]
     cxx.cflags += [
       '-pipe',
       '-fno-strict-aliasing',
       '-Wall',
+      '-Werror',
       '-Wno-unused',
       '-Wno-switch',
       '-Wno-array-bounds',
-      '-msse',
+      '-Wno-unknown-pragmas',
+      '-Wno-dangling-else',
       '-fvisibility=hidden',
     ]
+    if cxx.target.arch in ['x86', 'x86_64']:
+      cxx.cflags += ['-msse']
```

Comment:
Why was the flag -msse moved out, from above, if you're going to add it anyways regardless of the architecture ?